### DISCUSSION TITLE
Hover Docs (Noice)

### DISCUSSION THREAD

**Discussion by karimlevallois** [LazyVim/LazyVim#2926] [Q&A] [answered]
Hey everyone!

Is there a way to make the hover popups transparent the same as the cmp ones?

<img width="1680" alt="Screenshot 2024-04-05 at 12 46 24" src="https://github.com/LazyVim/LazyVim/assets/80129273/ef80ef66-611d-4695-bc5a-f4a7dca3ace0">

<img width="1680" alt="Screenshot 2024-04-05 at 12 46 34" src="https://github.com/LazyVim/LazyVim/assets/80129273/2b1c275a-a360-4d4f-a93d-8004a5e7fed3">

Many thanks

**Comment by rubiin** [comment]
@karimlevallois You can use this plugin https://github.com/soulis-1256/eagle.nvim . I think this is what you are asking

**Comment by dpetka2001** [comment]
```lua
  {
    "noice.nvim",
    opts = {
      presets = {
        lsp_doc_border = true,
      },
      lsp = {
        hover = {
          opts = {
            win_options = {
              winblend = 100,
            },
          },
        },
      },
    },
  },
```
Should do what you want. Play around with the value of `winblend` for how transparent you want it to be.

**Reply by karimlevallois** [reply]
Awesome thanks! Do you also know how to change the colour of the background so it's not so dark?

**Reply by dpetka2001** [reply]
By setting it transparent with `winblend` I don't believe it will have background color. Try it to see how it behaves.

**Comment by karimlevallois** [comment, marked as answer]
Got it working with a combination of the `winblend = 10`, along with `vim.cmd [[autocmd! ColorScheme * highlight NormalFloat guibg=bg]]`

<img width="1680" alt="Screenshot 2024-04-05 at 19 09 40" src="https://github.com/LazyVim/LazyVim/assets/80129273/db93c71b-99cc-4c57-a4b0-1b705ac47cbf">
